ant/src/org/jetbrains/kotlin/ant/Kotlin2JvmTask.kt

```diff
@@ -73,4 +79,48 @@ class Kotlin2JvmTask : KotlinCompilerBaseTask() {
         if (noReflect) args.add("-no-reflect")
         if (includeRuntime) args.add("-include-runtime")
     }
+
+    override fun execute() {
+        if (!fork)
+            super.execute()
+        else {
+            exec()
+        }
+    }
+
+    private fun exec() {
+        val javaHome = System.getProperty("java.home")
+        val javaBin = javaHome + separator + "bin" + separator + "java"
+        val classpath = System.getProperty("java.class.path")
+        var redirector = Redirector(this)
+
+        fillArguments()
+
+        val command = ArrayList<String>()
+        command.add(javaBin)
+        command.addAll(cmdl.vmCommand.arguments) // jvm args
+        command.add("-cp")
+        command.add(classpath + pathSeparator + KotlinAntTaskUtil.compilerJar.canonicalPath)
```

Comment:
Is `classpath` really necessary? If I understand correctly, it's the classpath of the currently-running Ant task, which is not needed for the compiler to work

Reply:
It seems a mistake and have been removed by now.